examples/api/engineering_formatter.py

```diff
@@ -14,13 +14,29 @@
 # Fixing random state for reproducibility
 prng = np.random.RandomState(19680801)
 
-fig, ax = plt.subplots()
-ax.set_xscale('log')
-formatter = EngFormatter(unit='Hz')
-ax.xaxis.set_major_formatter(formatter)
-
+# Create artificial data to plot.
+# The x data span over several decades to demonstrate several SI prefixes.
 xs = np.logspace(1, 9, 100)
-ys = (0.8 + 0.4 * prng.uniform(size=100)) * np.log10(xs)**2
-ax.plot(xs, ys)
+ys = (0.8 + 0.4*prng.uniform(size=100))*np.log10(xs)**2
```

Comment:
Huh, I guess it is an aftermath of my windy rebase, during which I had to take care of conflicts with another PR. I will go back to the previous spacing.